includes/admin/class-wp-job-manager-settings.php

```diff
@@ -303,6 +303,15 @@ protected function init_settings() {
 								'url'   => __( 'Website URLs only', 'wp-job-manager' ),
 							],
 						],
+						[
+							'name'       => 'job_manager_show_agreement_job_submission',
+							'std'        => '0',
+							'label'      => __( 'Agreement checkbox', 'wp-job-manager' ),
```

Comment:
I wonder if `Terms and conditions checkbox` makes more sense? What do you think?

Reply:
https://github.com/Automattic/WP-Job-Manager/pull/2155/commits/ea513f367b0e59288a60c7272c5b7ea9d450f545